src/DrTests/DrTests.class.st

```diff
@@ -420,11 +437,14 @@ DrTests >> updatePackagesList [
 { #category : #private }
 DrTests >> updatePackagesListLabel [
 	self packagesListLabel
-		label: (String streamContents: [ :s |
-			s
-				<< 'Packages (';
-				<< self packagesSelected size asString ;
-				<< ' selected):' ])
+		label:
+			(String
```

Comment:
Same here, extract the String building in a method.

Reply:
I am not sure since this method only update the label, but maybe I can use one method for building both Strings, I will do it